litmus-portal/frontend/src/ReduxRoot.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import ReduxRoot from './ReduxRoot';
+import '@testing-library/jest-dom';
+
+describe('<ReduxRoot />', () => {
+  it('renders', async () => {
```

Comment:
This is not a useful test...for trying out it's fine but should be removed before merge...
Try running test coverage and look for untested logic...if you want a real testcase

Reply:
Yes, this was for trying out!